src/platform/sdl2/main.cpp

```diff
@@ -463,7 +463,53 @@ void inputUpdate() {
     }
 }
 
+void print_help(int argc, char **argv) {
+    
+    printf("%s [OPTION]\nA open source re-implementation of the classic Tomb Raider engine.\n",
```

Comment:
argv[0]?

Reply:
Not sure what the objection is here, argv[0] contains the name of the binary with its calling path ie ./openlara or whatever the binary is called.